Kitodo/src/main/webapp/newpages/ProjekteBearbeiten.xhtml

```diff
@@ -1,3 +1,4 @@
+<?xml version="1.0" encoding="ISO-8859-1"?>
```

Comment:
It didn't work with UTF-8.
The problem is fixed with this pull request. I'm open for other suggestions, otherwise I'm going to merge this version.